packages/cli/src/user-management/email/templates/workflowShared.mjml

```diff
@@ -0,0 +1,17 @@
+<mjml>
+	<mj-include path="./_common.mjml" />
+	<mj-body>
+		<mj-section>
+			<mj-column>
+				<mj-text font-size="24px" color="#ff6d5a"> A workflow has been shared with you </mj-text>
+			</mj-column>
+		</mj-section>
+		<mj-section background-color="#FFFFFF" border="1px solid #ddd">
```

Comment:
Not familiar with MJML, maybe there's a way to extract common styles to `_common` and use CSS classes instead?